AboutIOFDev.rdf

```diff
@@ -1,37 +1,48 @@
-	xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
-	xmlns:iof-av="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/"
-	xmlns:dcterms="http://purl.org/dc/terms/">
-	
-  	<owl:Ontology rdf:about="https://spec.industrialontologies.org/ontology/AboutIOFDev/">
-		<rdfs:label xml:lang="en-US">About IOF Development</rdfs:label>
-		<dcterms:abstract>This ontology defines the scope of development maturity level IOF ontologies.</dcterms:abstract>
-		<dcterms:license rdf:datatype="&xsd;anyURI">http://opensource.org/licenses/MIT</dcterms:license>
-		<dcterms:title>Industrial Ontology Foundry (IOF) Ontology - Development</dcterms:title>
-		<iof-av:copyright>Copyright (c) 2022, 2023, Open Applications Group</iof-av:copyright>
-		<owl:versionIRI rdf:resource="https://spec.industrialontologies.org/ontology/202301/AboutIOFDev/"/>
-		<iof-av:maturity rdf:resource="&iof-av;Provisional"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/Core/"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/commonstocoremapping/meta/MappingAnnotationVocabularyToCommons/"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/commonstocoremapping/MappingCommonsToIOF/"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/maintenance/MaintenanceReferenceOntology/"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/supplychain/SupplyChain/"/>
-	</owl:Ontology>
+  xmlns:dct="http://purl.org/dc/terms/"
+  xmlns:owl="http://www.w3.org/2002/07/owl#"
+  xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+  xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+  xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+  xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+  xmlns:iof-av="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/"
+  xmlns:dcterms="http://purl.org/dc/terms/">
+  
+    <owl:Ontology rdf:about="https://spec.industrialontologies.org/ontology/AboutIOFDev/">
+    <rdfs:label xml:lang="en-US">About IOF Development</rdfs:label>
+    <dcterms:abstract>This ontology defines the scope of development maturity level IOF ontologies.</dcterms:abstract>
+    <dcterms:license rdf:datatype="&xsd;anyURI">http://opensource.org/licenses/MIT</dcterms:license>
+    <dcterms:title>Industrial Ontology Foundry (IOF) Ontology - Development</dcterms:title>
+    <iof-av:copyright>Copyright (c) 2022, 2023, Open Applications Group</iof-av:copyright>
+    <owl:versionIRI rdf:resource="https://spec.industrialontologies.org/ontology/202301/AboutIOFDev/"/>
+    <iof-av:maturity rdf:resource="&iof-av;Provisional"/>
+
+    <!-- load from the cache -->
+    <owl:imports rdf:resource="https://spec.industrialontologies.org/iof/ontology/cache_external_ontologies/latest/cache/bfo/2020/bfo.rdf"/>
```

Comment:
I would suggest using the actual IRI of the ontology here, i.e., `http://purl.obolibrary.org/obo/bfo.owl` and then "redirecting" it to the cache using an entry in https://github.com/iofoundry/ontology/blob/master/catalog-v001.xml.
Then software like Protege first will attempt resolve the IRI and if this fails, it will get it from the location specified in catalog-v001.xml. I.e., this additional entry may look like:

`<uri id="289c79d8-24a7-4ad6-a7d0-49da596ebe07" name="http://purl.obolibrary.org/obo/bfo.owl" uri="./cache/bfo/2020/bfo.rdf"/>`

Similar remarks apply to other import statements related to the "cached" ontologies.

Reply:
We require that all external ontologies are stored in the cache directory. 

We will sync and maintain the ontologies.

Reply:
@mereolog There have been issues with BFO access, so the decision was made to cache all external ontologies for IOF to guarantee that the versions used for the IOF are those that have been validated. The validated ones are in the cache, so that's what the about file is describing in this case. This does add maintenance overhead, but it ensures that the ontologies all work for anyone using IOF.